tests/gtest/avifsampletransformtest.cc

```diff
@@ -28,7 +28,7 @@ class AvifExpression : public avifSampleTransformExpression {
     token.type = AVIF_SAMPLE_TRANSFORM_INPUT_IMAGE_ITEM_INDEX;
     token.inputImageItemIndex = inputImageItemIndex;
   }
-  void AddOperator(uint8_t op) {
+  void AddOperator(avifSampleTransformTokenType op) {
```

Comment:
Thank you for the pull request. I am worried that this did not fix the compiler warning. (Compiler warnings are not enabled on tests yet. Sorry about the confusion.) I will test this using a draft pull request of mine later (https://github.com/AOMediaCodec/libavif/pull/2318).

The warning we need to fix is:

```
warning C4244: 'argument': conversion from 'const avifSampleTransformTokenType' to 'uint8_t', possible loss of data
```

Since `token.type` is a uint8_t, the assignment `token.type = op;` is now a conversion from 'const avifSampleTransformTokenType' to 'uint8_t', so I am worried that MSVC will warn about the assignment.

Reply:
Sorry, it was an oversight on my end. Fixed in https://github.com/AOMediaCodec/libavif/pull/2331.